core/templates/dev/head/forms/formBuilder.js

```diff
@@ -627,6 +641,10 @@ oppia.factory('rteHelperService', [
           el.attr(
             $filter('camelCaseToHyphens')(attrName) + '-with-value',
             oppiaHtmlEscaper.objToEscapedJson(customizationArgsDict[attrName]));
+          if (componentDefn.name === 'image' && attrName === 'width') {
```

Comment:
@seanlip As Kevin said, it's necessary to have a special case to get the preview size right in the editor, for now. If getting the preview size is not a big deal then we can remove this case.

It should be fairly temporary, once we switch to CKEditor and render the component directives directly there should not be a need for this anymore.